### DISCUSSION TITLE
@types/selectize : Missing methods clearOptionGroups, removeOptionGroup

### DISCUSSION THREAD

**Discussion by AndreEckner** [DefinitelyTyped/DefinitelyTyped#69336] [Issues with a @types package]
In types/selectize there are methods missing in the Interface Selectize.IApi<T, U>:

- clearOptionGroups(id)
- removeOptionGroup()
- addItems()

They are officially documented here:
https://selectize.dev/docs/API/selectize#removeoptiongroupid
https://selectize.dev/docs/API/selectize#clearoptiongroups
https://selectize.dev/docs/API/selectize#additemsvalues-silent

[a comment by an automated account is not shown]

**Reply by AndreEckner** [reply]
Okay, I changed it.

**Comment by ghost** [comment]
Fixed in #69378
